src/settings/MappingProfiles/detailsSections/edit/MappingMARCBibDetails.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { MARCTable } from '../../../../components/MARCTable';
+import { OverrideProtectedFieldsTable } from './MARCDetailsSection';
+
+import {
+  FIELD_MAPPINGS_FOR_MARC,
+  marcFieldProtectionSettingsShape,
+} from '../../../../utils';
+
+export const MappingMARCBibDetails = ({
+  marcMappingDetails,
+  fieldMappingsForMARCField,
+  marcFieldProtectionFields,
+  mappingMarcFieldProtectionFields,
+  setReferenceTables,
+}) => {
+  const marcTableFields = marcMappingDetails ||
```

Comment:
It looks like this variable is no longer needed because `marcMappingDetails` is defined in the default props.